lib/mcp-api-handler.ts

```
@@ -238,10 +270,11 @@ export function initializeMcpApiHandler(
         `requests:${sessionId}`,
         JSON.stringify(serializedRequest)
       );
-      console.log(`Published requests:${sessionId}`, serializedRequest);
+      logger.info(`Published request to channel`, { sessionId, requestId });
 
       let timeout = setTimeout(async () => {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'timeout' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)